include/mqtt/log.hpp

```diff
@@ -122,7 +122,7 @@ BOOST_LOG_ATTRIBUTE_KEYWORD(address, "MqttAddress", void const*)
 
 #if !defined(MQTT_ADD_VALUE)
 
-#define MQTT_ADD_VALUE(name, val) boost::log::add_value((name), (val))
+#define MQTT_ADD_VALUE(name, val) boost::log::add_value((MQTT_NS::name), (val))
```

Comment:
I don't understand why you added MQTT_NS:: to "name" ?

Reply:
In some of the log code below, this has a strange outcome.

MQTT_ADD_VALUE(address, spep.get()) -> boost::log::add_value(MQTT_NS::address, spep.get())

Reply:
It is the same reason as MQTT_LOG(chan, sev) 's sev.
https://github.com/redboltz/mqtt_cpp/blob/master/include/mqtt/log.hpp#L118